src/functions/getPosts/index.js

```diff
@@ -9,15 +9,50 @@
  */
 
 import r from 'rethinkdb'
+import Twitter from 'twitter'
 import { DB } from '../../db'
 
 export const method = 'GET'
 export const path = '/api/getPosts'
 
+function getTweets () {
```

Comment:
While you're making other changes, just have `location` be an input to the function